class ItemNotFoundError(Exception):
    pass


class ItemExistsError(Exception):
    pass

